docs/partial_source/design/building_blocks.rst

```diff
@@ -374,6 +376,39 @@ As an example, the following 3 pieces of code all compile to the exact same comp
    :align: center
    :width: 75%
 
+Now, let's take another example, this time setting the desired backend as tensorflow as tf.
```

Comment:
we can't compile `torch` code to `tensorflow` without using the transpiler, the rightmost example below does not make sense.

Reply:
oops, that's my bad. I forgot to change those functions to tf. I'll do that right away